prebuild/index.js

```diff
@@ -34,7 +36,7 @@ const targets = [
   { version: '18.0.0', abi: '108' },
   { version: '19.0.0', abi: '111' },
   { version: '20.0.0', abi: '115' }
-].filter(target => semver.satisfies(target.version, NODE_VERSIONS))
+].filter(target => semver.satisfies(target.version, `>=${lowestVersion}`))
```

Comment:
Ah yeah that makes sense. That's why this was a range in the first place. Could it maybe stay a range, and then the tests will just return early (with success) when the version was not in the range instead of skipping the job entirely?

Reply:
That would be a bunch more complicated because for every job we'd need a step to parse the semver range in JS, store the result in an actions output, and then put conditions on every subsequent step to skip it. It's possible, but a lot more code. 😐

Reply:
As discussed yesterday, using a range as 2 environment variables (or just one for now since only a minimum is immediately needed) would actually be much easier.